INSTALL

```diff
@@ -0,0 +1,169 @@
+           Netatalk Installation
+           =====================
+
+Quick start
+===========
+
+Netatalk can be built by using Meson or GNU Autotools.
+The meson build system is the recommended build system for the master
+branch, autotools support may be be removed at some point in the future.
```

Comment:
Shall we aim for a deprecation policy and window? E.g. if no major breakages or loss of compatibility with a tier 1 OS, we deprecate autotools in v3.3

Reply:
Yes, I agree. The dbus devel team window for withdrawing autotools support was about 1 year. Meson was introduced in version 1.15.0 (22/9/22) and autotools was removed in version 1.15.8 (21/8/23). Deprecation warnings were posted in NEWS.